spec/support/factory_girl.rb

```diff
@@ -2,6 +2,7 @@
 
 FactoryGirl.define do
   factory :game do
+    user_id 4
```

Comment:
instead of this line, just change it to `user`. That is the same as `association: user` which creates a user object each time a game object is created.